src/BasicRum/Filters/Primary/OperatingSystem.php

```diff
@@ -15,4 +15,24 @@ public function getPrimarySearchFieldName(): string
     {
         return 'os_id';
     }
+
+    public function getSchema(): array
```

Comment:
Do you mean to recreate the whole Filters logic in src/BasicRum/DiagramSchema/* ?
Because the idea was that each filter can add its own piece of json schema, so that when you add new filter it will bring as well its own piece of final json schema. 

Let me know how you see this should work

Reply:
Hello @korzol 

Exactly, put the needed logic under src/BasicRum/DiagramSchema/ 

Think about it that we are implementing a component/plugin in our system. It should be as much as possible decoupled from the rest classes of the system.